grammars.yml

```diff
@@ -330,6 +330,9 @@ vendor/grammars/language-shellscript:
 - text.shell-session
 vendor/grammars/language-yaml:
 - source.yaml
+vendor/grammars/language-jsoniq
```

Comment:
There's a colon missing here (I'm surprised because you shouldn't have to edit this file manually).